dayFour/sample.java

```diff
@@ -0,0 +1,28 @@
+package dayFour;
+
+public class sample{
+	public void mth1() throws Exception {
+		mth2();
+		System.out.println("caller");
+	}
+	public void mth2() throws Exception {
+		try {
+			throw new Exception("custom exception");
+//			return;
+		}
```

Comment:
Can you add the catch statement for this exception and add your observations?

Reply:
I have added the catch statement and after adding it, the code after the try-catch executes without stopping